Simulator/components/CollapsibleTable.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { Component, PropTypes } from 'react';
+import { Table, Collapse, Button } from 'react-bootstrap';
+
+class CollapsibleTable extends Component {
```

Comment:
It might be nice to just put a quick comment describing each of the components. It took me a little bit to figure out where everything fits together just reading the code